frontend/web/components/metadata/AddMetadataToEntity.tsx

```diff
@@ -0,0 +1,251 @@
+import React, { FC, useEffect, useState } from 'react'
```

Comment:
Is this component responsible for entering metadata inside an entity e.g. creating a flag?

If so, does it pre-populate the entity with required items so that they don't have to find them via add metadata? I think this comment was in previous reviews.

Reply:
This component displays a list of metadata, each with an input field where the values for the metadata will be entered.

Reply:
Ok so all meta fields associated gets added regardless of required? Makes sense if so